aspnetcore/grpc/health-checks/Startup.cs

```diff
@@ -0,0 +1,20 @@
+#region snippet_1
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddGrpc();
+    services
+        .AddGrpcHealthChecks()
+        .AddCheck("Sample", () => HealthCheckResult.Healthy());
+}
+
+public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+{
+    app.UseRouting();
+    
+    app.UseEndpoints(endpoints =>
+    {
+        endpoints.MapGrpcService<GreeterService>();
+        endpoints.MapGrpcHealthChecksService();
+    });
+}
+#endregion
```

Comment:
This is .NET 5 code.  Would you like us to update it to the new minimal hosting model. Do you need to support .NET 5? I can fork this branch and update it to .NET 6 if you like. Ditto for
ditto for 
```
services.AddGrpcHealthChecks(o =>
{
    o.Services.MapService("greet.Greeter", r => r.Tags.Contains("greeter"));
    o.Services.MapService("count.Counter", r => r.Tags.Contains("counter"));
});
```

Reply:
> This is .NET 5 code. Would you like us to update it to the new minimal hosting model. Do you need to support .NET 5? I can fork this branch and update it to .NET 6 if you like. Ditto for
> 
> ```
> services.AddGrpcHealthChecks(o =>
> {
>     o.Services.MapService("greet.Greeter", r => r.Tags.Contains("greeter"));
>     o.Services.MapService("count.Counter", r => r.Tags.Contains("counter"));
> });
> ```

Updating the Startup.cs to show .NET 5 and .NET 6 would be good

I don't think the inline piece of code needs to be updated. `AddGrpcHealthChecks` is generic to .NET 5 or 6.

Reply:
> I don't think the inline piece of code needs to be updated. AddGrpcHealthChecks is generic to .NET 5 or 6.
In the .NET 6 version, `services.AddGrpcHealthChecks(o =>` becomes `builder.Services.AddGrpcHealthChecks(o =>`

I'll fork this branch and create .NET 5 and 6 versions.

Reply:
See #24660, which merges into this branch.